styles.css

```diff
@@ -578,8 +588,9 @@ form input {
     background-image: url("./assets/contact\ form\ shapes-mobile@2x.png");
     background-size: 100%;
     background-repeat: no-repeat;
-    padding: 5px;
     padding: 30px;
+
+    /* width: 90%; */
   }
```

Comment:
Hello @Marlyn-Mayienga ✋

- [ ] You may please always remove commented code from your project. This is not considered to be good practice.